test/snippet/core/detail/all_of_pack.cpp

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
remove newline :nail_care:

Reply:
we always use an empty line between seqan3 and other library headers